routes.js

```diff
@@ -206,6 +206,12 @@ const sharedRoutes = {
       title: navigation.state.params.title,
     }),
   },
+  SearchUsersByLocation: {
```

Comment:
In order to share this screen for other places, I suggest to define it as `Search` and replace the one in `SearchStackNavigator` with `MainSearch`.